src/bounded_loop.rs

```diff
@@ -0,0 +1,156 @@
+use std::fmt;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::env;
+use std::collections::HashMap;
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::io::{BufReader, BufRead, Read};
+use std::fs::File;
+
+lazy_static!{
+    static ref MUTAGEN_LOOP_COUNT: HashMap<LoopId, usize> = {
+        let mut h = HashMap::new();
+
+        File::open("target/mutagen/loops.txt").map(|mut file| {
+            let reader = BufReader::new(file);
+
+            for l in reader.lines() {
+                let line = l.unwrap_or(String::from(""));
+                let splits: Vec<&str> = line.split(",").collect();
```

Comment:
I was thinking on adding `serde` and  `serde_json` as depdency. On one hand, I think that maybe it's too much, but on the other it's nicer to use. Do you have any preference?

Reply:
I also think it's a bit much. All we need here are <loop ID>, <count> pairs. Also a clean mutagen build is already in the 100s of megabytes, so I'm a bit wary of adding such a large dependency.